packages/vue/src/components/molecules/SfBanner/SfBanner.vue

```diff
@@ -1,6 +1,13 @@
 <template>
   <section class="sf-banner" :style="style" v-on="isMobile ? $listeners : {}">
-    <div class="sf-banner__container">
+    <SfButton
```

Comment:
Now we have a button inside another button :)

Reply:
I changed it to have a 'div' wrapper on desktop and 'a' or 'button' on mobile depending on passing the link prop or not. Wdyt @justyna-13 ?